### ISSUE TITLE
Display options stats for int fields should show sum across field, not count

### ISSUE THREAD

**Issue by brimoor** [voxel51/fiftyone#507]
In the screenshot below, the `*_iou_0_75` fields are all `IntField`s. The counts currently show the number of fields that have a vale for the field, but, for integer fields, the user would really like to see the sum of the values across the dataset.

In the example below, this would correspond to the number of true positives/false positives across the dataset.

<img width="1696" alt="Screen Shot 2020-09-04 at 7 58 15 AM" src="https://user-images.githubusercontent.com/25985824/92237184-edbe8000-ee84-11ea-8c6d-35a167e0371f.png">

**Comment by lethosor** [comment]
I don't think we can make this generalization about all integer fields. For consistency with other scalar fields (strings, floats, etc.), it would be less confusing to use the same counting logic for all types, particularly when fields are absent (not just set to 0) for some samples.
Maybe a subtype of `IntField` (`CountField`?) could use the behavior you described, though.
